ingest/checkpoint_change_reader.go

```
@@ -483,6 +414,14 @@ LoopBucketEntry:
 						return false
 					}
 				}
+			} else if entry.Type == xdr.BucketEntryTypeInitentry && unique {
+				// we can remove the ledger key because we know that it's unique in the ledger
+				// and cannot be recreated
+				err := r.tempStore.Remove(h)
+				if err != nil {
+					r.readChan <- r.error(errors.Wrap(err, "Error removing key from tempStore"))
```

is it worth err'ing out, could it suffice to silently continue and log instead?